### PR TITLE
chore(RN): add migration release note for 2.45.0

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

File: md/migration-tool.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ to the one delivered with the latest version of Bonita.
 This is due to the fact that improvements in any version of the Migration Tool can affect all supported versions of Bonita.
 This changelog must be read before [migrating to a newer version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md).
 
+chore(RN): add migration release note for 2.44.0
```
Comment (educhastenier) [inline md/migration-tool.md]: ???!!!!
```suggestion
```

**Comment by educhastenier** [comment]
:warning: just wait for the Migration Tool release before merging this PR.

**Comment by tnguyen1** [comment]
retest this please

File: md/migration-tool.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ to the one delivered with the latest version of Bonita.
 This is due to the fact that improvements in any version of the Migration Tool can affect all supported versions of Bonita.
 This changelog must be read before [migrating to a newer version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md).
 
+## 2.44.1 - September 5th, 2019 (Bonita 7.9.3)
```
Comment (baptistemesta) [inline md/migration-tool.md]:
```suggestion
## 2.45.0 - October 3rd, 2019 (Bonita 7.9.4)
```

File: md/migration-tool.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ to the one delivered with the latest version of Bonita.
...
+This version of the migration tool migrates Bonita up to version 7.9.3.
```
Comment (baptistemesta) [inline md/migration-tool.md]:
```suggestion
This version of the migration tool migrates Bonita up to version 7.9.4.
```

**Comment by baptistemesta** [comment]
⚠️ should change commit name on merge

**Pull request merged**